### PR TITLE
Xonsh shell has jedi extension

### PR DESCRIPTION
**Pull request by anki-code**
Hello! Thank you for jedi! Xonsh shell has jedi extension.

### CODE REVIEW

**Comment by davidhalter** [comment]
Can you update docs/docs/usage.rst as well?

**Comment by anki-code** [comment]
@davidhalter updated!

File: docs/docs/usage.rst [hunk 1]
```diff
@@ -97,6 +97,16 @@ and many more!
 
 .. _repl-completion:
```
Comment (davidhalter) [inline docs/docs/usage.rst:98]: This should be below the next title :)

Comment (anki-code) [reply docs/docs/usage.rst]: Done :)

**Pull request merged**